services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/attribute/OffsetDateTimeAsStringAttributeConverter.java

```diff
@@ -58,9 +58,9 @@
  * </ul>
  *
  * <p>
- * This converter can read any values written by itself, {@link InstantAsIntegerAttributeConverter},
- * {@link InstantAsStringAttributeConverter}, or {@link ZonedDateTimeAsStringAttributeConverter}. Values written by
- * {@code Instant} converters are treated as if they are in the UTC time zone (and an offset of 0 seconds will be returned).
+ * This converter can read any values written by itself, {@link InstantAsStringAttributeConverter},
+ * or {@link ZonedDateTimeAsStringAttributeConverter}. Values written by {@code Instant} converters
+ * are treated as if they are in the UTC time zone (and an offset of 0 seconds will be returned).
```

Comment:
What about if the ZonedDateTime uses a named Zone, does this not break the data-loss principle?